genuser.sh

```
@@ -1,2 +1,2 @@
 #!/usr/bin/env bash
-docker run -v $HOME/cyberpot:/data --entrypoint bash -it -u $(id -u):$(id -g) khulnasoft/cyberpotinit:24.04.1 "/opt/cyberpot/bin/genuser.sh"
+docker run -v $HOME/cyberpot:/data --entrypoint bash -it -u $(id -u):$(id -g) khulnasoft/cyberpot-init:24.04.1 "/opt/cyberpot/bin/genuser.sh"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)